changelogs/3.3.2.md

```diff
@@ -0,0 +1,193 @@
+# 3.3.2
+
+Date: 2025-04-10
+Tag: 3.3.2
+
+## Overview
+
+Tarantool 3.x is the recommended release series. Users of Tarantool 2.11 are
+encouraged to update to the latest 3.x release.
+
+This release introduces 4 improvements and resolves 46 bugs since 3.3.1.
```

Comment:
We shouldn't add features into bugfix releases. We allow tooling improvements, sure, but let's advertise it differently from tarantool features.

I propose to drop 'introduces 4 improvements' from the overview, move the tooling section to the bottom, make it the second-level section 'tools' instead of 'feature/tools'. We can even merge it into the 'bugfix/tools', because these changes don't look very different by its sense.

To sum up: it shouldn't look like we introduce features in a bugfix release.

Reply:
I had similar reasoning to yours, but decided to leave it under "Functionality added or changed" section.

But ok, I don't mind moving it to bugfixes

Reply:
Resolving, the fix is applied: https://github.com/tarantool/tarantool/pull/11368#issuecomment-2794966184